# Eric J. Ma

## Contact Information

- Phone: 857-209-1375
- Email: http://shortwhale.com/ericmjl
- Website: https://ericmjl.github.io/
- Location: North Quincy, MA
- GitHub: [ericmjl](https://github.com/ericmjl)

## Skills


- Languages and Tools: Python, HTML, CSS, bash scripting, git, Illustrator, Indesign, Photoshop

- Packages: Pandas, PyMC3 (Bayesian statistics), Flask, scikit-learn, scikit-image, Keras, NetworkX, Mesa

- ML/Stats: Network science, variational autoencoders, random forests, Bayesian inference, graphical models, agent-based models

- Life Sciences: Microbiology, virology, biochemistry, molecular biology



## Experience


**Novartis Institutes for Biomedical Research (NIBR), Investigator II, Cambridge MA** (September 2018-Present)
- Investigator in the Scientific Data Analysis (SDA) team reporting to [Holger Hoefling](https://www.linkedin.com/in/holger-hoefling-8418905).
- Led an initiative to characterize the performance of message passing neural networks.
- Developed a hierarchical Bayesian 4-parameter dose response model to aid project team in compound selection.
- Currently developing a deep Bayesian optimization pipeline to accelerate protein engineering cycles.
- Co-hosted a machine learning session with [Sivakumar Gowrisankar](https://www.linkedin.com/in/sivakumar-gowrisankar-b71a016/) at the internal Data in Drug Discovery Day (D4).
- Co-organized an internal course with [Yuan Wang](https://www.linkedin.com/in/yuan-wang-王元-0b19a822/) and [Laszlo Urban](https://www.linkedin.com/in/laszlo-urban-4444974/) on machine learning for NIBR colleagues.
- Led a deep learning workshop at our NIBR Shanghai office.


**Novartis Institutes for Biomedical Research (NIBR), Investigator I, Cambridge MA** (September 2017-Present)
- Investigator in the Scientific Data Analysis (SDA) team reporting to [Mark Borowsky](https://www.linkedin.com/in/borowsky/).
- Performed internal consulting projects and expanded the SDA Statistical Learning initiative with colleagues.
- Developed parameterized Bayesian agent-based models of internal project portfolio for scenario planning purposes, using [PyMC3](http://docs.pymc.io) and [Mesa](https://github.com/projectmesa/mesa).
- Authored custom deep learning package to train graph convolutional neural networks to predict structural determinants of RNA cutting, with one paper currently in writing.
- Assisted in mentoring two interns, [Stacy Meichle](https://www.linkedin.com/in/stacy-meichle/) (Computer Aided Drug Discovery) and [Fritz Lekschas](https://www.linkedin.com/in/flekschas/) (SDA).


**Insight Data Science, Health Data Science Fellow, Boston MA** (June 2017-August 2017)
- Built [Flu Forecaster](https://fluforecaster.herokuapp.com), a machine learning-powered system that forecasts flu sequences six months out, to better prepare for manufacturing of vaccine strains.
 - Implemented a variational autoencoder (deep learning model) to learn a continuous representation of 14,455 influenza hemagglutinin protein sequences, and trained a Gaussian process model on the continuous representation to predict new flu sequences.
 - Developed interactive blog post using Flask and Bootstrap, and deployed to Heroku and GitHub.
 - Led peer workshops on [web development](https://github.com/ericmjl/minimal-flask-example), deep learning and [code style](https://github.com/ericmjl/pycodestyle-tutorial).


**Massachusetts Institute of Technology, ScD Candidate, Cambridge MA** (August 2011-May 2017)
- Developed a scalable, network-based phylogenetic heuristic algorithm for detecting reassortant influenza viruses using 18,632 fully sequenced virus genomes that improved our capacity to detect reassortment events by two orders of magnitude.
- The algorithm was used in a lead-author study (published in [PNAS](http://www.pnas.org/content/113/19/5335.full)) providing systematic evidence that genome shuffling is important for host switching, and a co-authored study (published in [Ecology Letters](http://onlinelibrary.wiley.com/doi/10.1111/ele.12629/abstract)) that showed that reassortment is a strategy for viral gene persistence in wild animal reservoirs.
- Contributed [reproducible data analysis](https://github.com/runstadler-lab/Seal-H3N8-Image-Analysis) for colleagues through fluorescent image quantification and genomic analysis in studies that refuted binding properties of novel influenza viruses.
- Performed [Bayesian statistical modelling](https://github.com/ericmjl/mia-stats/blob/master/sterilization/sterilization.ipynb) for colleagues testing the efficacy of phone sterilization tools.
- Delivered tutorials and talks on Network Analysis and Bayesian statistical methods at annual Python conferences, including PyCon, SciPy, and PyData.


**The University of British Columbia, Undergraduate Researcher, Vancouver BC** (June 2006-May 2010)
- Investigated the role of T cells in intestinal inflammation and fibrosis in a Salmonella typhimurium-induced model of gut inflammation, leading to First Prize Poster ([Rising Stars of Research 2008](https://www.evernote.com/l/AANJS_A9iKZAMLGYqJDduTxiiI2ZNf43zHM)) and Best Talk (Multidisciplinary Undergraduate Research Conference 2009) awards.
- Co-founded the [first UBC International Genetically Engineered Machines (iGEM) team](http://2009.igem.org/Team:British_Columbia), where we achieved a Gold status standing in our first year of competition.
- Served as a [Peer Academic Coach](https://science.ubc.ca/students/spac), guiding first- and second-year students towards academic success through building good habits on learning strategies and time management.




## Education


**Sc.D.**, Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge MA (August 2011-May 2017)

**B.Sc.**, Integrated Sciences, The University of British Columbia, Cambridge MA (June 2006-May 2010)



## Publications


- Long-term colonization dynamics of Enterococcus faecalis in implanted devices in research macaques *Applied and Environmental Microbiology* (2018). [link](https://aem.asm.org/content/early/2018/07/09/AEM.01336-18)

- Evaluation of 6 Methods for Aerobic Bacterial Sanitization of Smartphones *Journal of the American Association for Laboratory Animal Science* (2018). [link](https://www.ingentaconnect.com/content/aalas/jaalas/2018/00000057/00000001/art00004)

- Evaluation of 6 Methods for Aerobic Bacterial Sanitization of Smartphones *Journal of the American Association for Laboratory Animal Science* (2017). [link](https://www.ncbi.nlm.nih.gov/pubmed/29402348)

- Reassortment of influenza A viruses in wild birds in Alaska before H5 clade 2.3. 4.4 outbreaks *Emerging Infectious Diseases* (2017). [link](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5367406/)

- A real-time surveillance dashboard for monitoring viral phenotype from sequence *International Journal of Infectious Diseases* (2016). [link](http://www.ijidonline.com/article/S1201-9712(16)31510-7/abstract)

- Evidence of seasonality in a host-pathogen system: Influenza across the annual cycle of wild birds *Integrative and Comparative Biology* (2016). [link](http://www.sicb.org/meetings/2016/schedule/abstractdetails.php?id=502)

- A point mutation in the polymerase protein PB2 allows a reassortant H9N2 influenza isolate of wild-bird origin to replicate in human cells *Infection, Genetics and Evolution* (2016). [link](https://www.sciencedirect.com/science/article/pii/S156713481630137X)

- New England harbor seal H3N8 influenza virus retains avian-like receptor specificity *Scientific Reports* (2016). [link](https://www.nature.com/articles/srep21428)

- Ecosystem interactions underlie the spread of avian influenza a viruses with pandemic potential *PLOS Pathogens* (2016). [link](http://onlinelibrary.wiley.com/doi/10.1111/ele.12629/abstract)

- Transmission of influenza reflects seasonality of wild birds across the annual cycle *Ecology Letters* (2016). [link](http://onlinelibrary.wiley.com/doi/10.1111/ele.12629/abstract)

- Reticulate evolution is favored in influenza niche switching *PNAS* (2016). [link](http://www.pnas.org/content/113/19/5335.full)

- Genetic characterization of a rare H12N3 avian influenza virus isolated from a green-winged teal in Japan *Virus Genes* (2015). [link](https://link.springer.com/article/10.1007/s11262-014-1162-9)
